docs/installation.rst

```diff
@@ -76,6 +79,22 @@ a. Set up ``udev`` rules for appropiate permission to ``/dev/ttyACM*`` in Linux
 
     1. Create a file named ``setup_udev_rule.sh`` and include the following script
 
+Mac
```

Comment:
Was this inserted in the wrong location? There should be the script to set permissions in Linux here (which is now below the added Mac section).

Looks a bit like something might've gone wrong in the merge?